unittests/CppInterOp/ScopeReflectionTest.cpp

```diff
@@ -952,3 +952,50 @@ TEST(ScopeReflectionTest, IncludeVector) {
   )";
   Interp->process(code);
 }
+
+TEST(ScopeReflectionTest, GetBinaryOperator) {
+  if (llvm::sys::RunningOnValgrind())
+    GTEST_SKIP() << "XFAIL due to Valgrind report";
```

Comment:
We can drop that as Aaron has added to the suppression file.

Reply:
The InterOp unitttests don't use a a suppression file, they are only for cppyy

Reply:
Ok. Then we can ignore this comment.